src/Sulu/Bundle/MediaBundle/Command/FormatCacheRegenerateFormats.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Command;
+
+use Sulu\Bundle\MediaBundle\Media\FormatManager\FormatManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+class FormatCacheRegenerateFormats extends Command
+{
+    protected static $defaultName = 'sulu:media:regenerate-formats';
+
+    /**
+     * @var Filesystem
+     */
+    private $fileSystem;
+
+    /**
+     * @var FormatManagerInterface
+     */
+    private $formatManager;
+
+    /**
+     * @var string
+     */
+    private $localFormatCachePath;
+
+    public function __construct(
+        Filesystem $filesystem,
+        FormatManagerInterface $formatManager,
+        string $localFormatCachePath
+    ) {
+        parent::__construct();
+
+        $this->fileSystem = $filesystem;
+        $this->formatManager = $formatManager;
+        $this->localFormatCachePath = $localFormatCachePath;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Loops over sulu image cache, and regenerates the existing images')
+        ;
```

Comment:
```suggestion
        $this->setDescription('Loops over sulu image cache, and regenerates the existing images');
```

Not sure why php-cs didn't catch did but `;` should be in the same line. Else it looks good from miy side.